build/build.proj

```diff
@@ -108,7 +108,7 @@
 
   </Target>
 
-  <Target Name="Build" DependsOnTargets="RestorePackages;BuildSolution;BuildNuGetPackages;Test" />
-  <Target Name="Rebuild" DependsOnTargets="RestorePackages;RebuildSolution;RebuildNuGetPackages;Test" />
+  <Target Name="Build" DependsOnTargets="RestorePackages;BuildSolution;BuildNuGetPackages;BuildModernVsixPackages;Test" />
```

Comment:
This will likely break microbuild. We specifically separated out building V3 VSIX and regular VSIX into separate targets as Microbuild had a bug where building them with the same msbuild invocation caused build failures. Unless, Microbuild has fixed this issue, this will break signed builds.